SmallCapitalLetters/src/main/java/SmallLetters.java

```diff
@@ -0,0 +1,7 @@
+public class SmallLetters implements TextFormatter{
```

Comment:
Zauważ, że klasy zazwyczaj nazywamy od rzeczowników np.: Calculator, String, Date. Ten interfejs w tym momencie bardzo fajnie CI pokazuje jak można by nazwać klasę - skoro mamy interfejs 'TextFormatter' to pytanie brzmi 'jaki TextFormatter?'

odpowiedź:
LowerCaseTextFormatter